python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_chose_trend.py

```diff
@@ -65,12 +65,10 @@
 ot.RandomGenerator.SetSeed(0)
 nTrain = 8
 Xtrain = ot.Uniform(xmin, xmax).getSample(nTrain).sort()
-Xtrain
```

Comment:
This line should not be removed because it prints the sample:
![image](https://github.com/openturns/openturns/assets/31351465/46d4f2ff-e280-47bc-8563-e6eab734c1d9)